user_manual/files/webgui/sharing.rst

```diff
@@ -40,6 +40,22 @@ with the user "pierpont", who can *share*, *edit*, *create*, *change*, and
 .. figure:: ../../images/files_page-2.png
    :alt: Sharing files.
 
+What Happens When Share Recipients Move Files and Folders?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If a share recipient has "**can edit**" privileges and moves files or folders out of the share, ownCloud restores a backup copy of the moved file/folder in the Deleted Files (Trash) of the share’s owner. 
```

Comment:
restores => stores ?

Reply:
hmmm, comme ci comme ça? Ok, I prefer "restores", but the other wording will have to be adjusted to fit. You're right.

Reply:
No, on reading it again, you're right.